sauce/features/general/import-notification/index.js

```diff
@@ -0,0 +1,72 @@
+import { Feature } from 'core/feature';
+// import * as toolkitHelper from 'helpers/toolkit';
+
+export class ImportNotification extends Feature {
+  injectCSS() { return require('./index.css'); }
+
+  constructor() {
+    super();
+    this.isActive = false;
+    this.importClass = 'import-notification';
+    this.invoke = this.invoke.bind(this);
+  }
+
+  willInvoke() {
+    if (this.settings.enabled !== '0') {
+      if (this.settings.enabled === '2') {
+        this.importClass += '-red';
+      }
+
+      // Hook transaction imports so that we can run our stuff when things change
+      ynab.YNABSharedLib.defaultInstance.entityManager._transactionEntityPropertyChanged.addHandler(this.invoke);
+    }
+  }
+
+  shouldInvoke() {
+    return this.settings.enabled !== '0';
+  }
+
+  invoke() {
+    if (!this.isActive) {
```

Comment:
not really sure what this guard is for, everything in `checkImportTransactions` seems synchronous so I'm not sure when this code would get hit where `isActive` is false?